### ISSUE TITLE
`pod install` fails on a fresh install of 0.35.0 with `dir_initialize` error.

### ISSUE THREAD

**Issue by chakrit** [CocoaPods/CocoaPods#3121]
By fresh install I mean a fresh install of ruby and cocoapods. Just cloned a project that uses cocoapods which worked fine before and then when trying to run `pod install --verbose` this is the result:

**CONSOLE OUTPUT:**

```
  Preparing

Analyzing dependencies

Updating spec repositories
/Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/2.1.0/pathname.rb:425:in `open': No such file or directory @ dir_initialize - /Users/chakrit/.cocoapods/repos (Errno::ENOENT)
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/2.1.0/pathname.rb:425:in `foreach'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/2.1.0/pathname.rb:425:in `children'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/cocoapods-0.35.0/lib/cocoapods/sources_manager.rb:63:in `all'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/cocoapods-0.35.0/lib/cocoapods/user_interface/error_report.rb:130:in `repo_information'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/cocoapods-0.35.0/lib/cocoapods/user_interface/error_report.rb:34:in `report'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/cocoapods-0.35.0/lib/cocoapods/command.rb:58:in `report_error'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/claide-0.7.0/lib/claide/command.rb:300:in `handle_exception'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/claide-0.7.0/lib/claide/command.rb:274:in `rescue in run'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/claide-0.7.0/lib/claide/command.rb:264:in `run'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/cocoapods-0.35.0/lib/cocoapods/command.rb:45:in `run'
	from /Users/chakrit/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/cocoapods-0.35.0/bin/pod:43:in `<top (required)>'
	from /Users/chakrit/.rbenv/versions/2.1.5/bin/pod:23:in `load'
	from /Users/chakrit/.rbenv/versions/2.1.5/bin/pod:23:in `<main>'
```

**INFO:**

```
$ ruby --version
ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-darwin14.0]
$ pod --version
0.35.0
```

Xcode is latest beta on dev.apple

**Comment by chakrit** [comment]
Manually `mkdir -p ~/.cocoapods/repos` fixes it. But cocoapods should really do that automatically isn't it?

**Comment by orta** [comment]
There's a bug with 0.35 that sometimes requires people to run `pod setup` before their first install. This is fixed in 0.36
